### PR TITLE
Add index on @owner_id

### PR DESCRIPTION
**Pull request by amstone326**
cross-request: https://github.com/dimagi/commcare-core/pull/553

### CODE REVIEW

**Comment by amstone326** [comment]
@damagatchi retest this please

File: app/src/org/commcare/models/database/user/models/AndroidCaseIndexTable.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public static void createIndexes(SQLiteDatabase db) {
 
     /**
      * Creates all indexes for this case.
-     * TODO: this doesn't ensure any sort of uniquenes, you should wipe constraints first
+     * TODO: this doesn't ensure any sort of uniqueness, you should wipe constraints first
```
Comment (wpride) [inline app/src/org/commcare/models/database/user/models/AndroidCaseIndexTable.java]: lol

**Comment by amstone326** [comment]
fyi @sheelio, this will be in 2.36

**Comment by amstone326** [comment]
@ctsims Does this feel worth of a release note or announcement to field-dev?

**Comment by amstone326** [comment]
Announce to field-dev: @owner_id is now an optimized case search parameter

**Pull request merged**